src/style.css

```
@@ -0,0 +1,28 @@
+html,
+body,
+div {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+div[class*="stars--"] {
```

```suggestion
stars {
```
when you would add class `stars` that I have mentioned, you would be able to use it there